websites/A/Animeflix/metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "326694667003363328",
+		"name": "Micotsu#2907"
+	},
+	"service": "Animeflix",
+	"description": {
+		"en": "Unlimited anime without ads!"
+	},
+	"url": "animeflix.live",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/4hiyxuW.png",
+	"thumbnail": "https://i.imgur.com/XmDNhW4.png",
```

Comment:
This thumbnail may be a bit small/low-quality, but I'm not sure if it *needs* to be replaced.